ansibledriver/ibm_cp4na_log_message.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+# This custom module is for printing some specified logs
+
+from ansible.module_utils.basic import AnsibleModule
+
+def run_module():
+    # defining available arguments/parameters a user can pass to the module
+    module_args = dict(
+        message_direction=dict(type='str', required=True, choices=['sent', 'received']),
+        external_request_id=dict(type='str', required=True),
+        message_data=dict(type='str'),
```

Comment:
I know there have been some late design changes, would it be possible to get the new `message_type` field in here before we merge this? Let me know if it's too late to add that.

Reply:
Sure. We have now added code for supporting `message_type` in the module. Code has been pushed.